lib/core/Canvas.js

```diff
@@ -859,6 +862,66 @@ Canvas.prototype.scroll = function(delta) {
   return { x: matrix.e, y: matrix.f };
 };
 
+/**
+ * Scrolls the viewbox to contain the given element.
+ * Optionally specify a padding to be applied to the edges.
+ *
+ * @param {Object} [element] the element to scroll to.
+ * @param {Object|Number} [padding=100] the padding to be applied. Can also specify top, bottom, left and right.
+ *
+ */
+Canvas.prototype.scrollToElement = function(element, padding) {
+  var defaultPadding = 100;
+  if (!padding) {
+    padding = {};
+  }
+  if (typeof padding === 'number') {
+    defaultPadding = padding;
+  }
+
+  padding = {
+    top: padding.top || defaultPadding,
+    right: padding.right || defaultPadding,
+    bottom: padding.bottom || defaultPadding,
+    left: padding.left || defaultPadding
+  };
+
+  var elementBb = getBoundingBox(element),
```

Comment:
Can we rename `elementBb` and other variables that represent bounding boxes to `elementBounds` and so on? We usually use that name. There's also a function [LayoutUtil#asBounds](https://github.com/bpmn-io/diagram-js/blob/develop/lib/layout/LayoutUtil.js#L57) that has this name.